src/api/zoops.ts

```diff
@@ -33,4 +33,29 @@ zoopsRouter.post("/", async (req, res, next): Promise<void> => {
     }
 })
 
+// DELETE /api/zoops/:id
+// TODO: Add auth middleware to check if user is logged in once it is available from merging of login endpoint branch
+zoopsRouter.delete(`/:id`, async (req, res, next) => {
+    const userId = Number(req.headers.id);
+    try {
+        const { id } = req.params
+        const zoop = await prisma.zoop.findFirstOrThrow({
```

Comment:
```suggestion
        const zoop = await prisma.zoop.findUniqueOrThrow({
```

Find first would not error if there are duplicate ids - recommend unique when looking up by ID

Reply:
Good point!